examples/react-app/src/pages/Home/index.tsx

```diff
@@ -21,9 +167,18 @@ const ScenePage = () => {
             </Container>
             <Container header={<Header>Scene</Header>}>
               <div style={{ height: '864px', position: 'relative' }}>
```

Comment:
you can move these inline style to a separate file.

Reply:
I am fine with this since it doesn't belong to this PR and should be addressed in a separate PR.

Reply:
I ended up digging on this because it seemed easy, and then found we'd moved our global css file from App.css to index.css as part of the Vite conversion so there is now some general CSS clean up in this to align with our css best practice and desire to have minimal styling on app-kit.